internal/appland/client.go

```diff
@@ -12,11 +12,29 @@ import (
 	"github.com/applandinc/appland-cli/internal/config"
 )
 
-type Client struct {
+type Client interface {
+	BuildUrl(paths ...interface{}) string
+	Context() *config.Context
+	CreateMapSet(app, org string, scenarios []string) (*CreateMapSetResponse, error)
+	CreateScenario(org string, scenarioData io.Reader) (*ScenarioResponse, error)
+	DeleteAPIKey() error
+	Login(login string, password string) error
+	TestAPIKey(apiKey string) (bool, error)
+	delete(url string, body io.Reader) (*http.Response, error)
+	get(url string, body io.Reader) (*http.Response, error)
+	newAuthRequest(method, url string, body io.Reader) (*http.Request, error)
+	post(url string, body io.Reader) (*http.Response, error)
+}
+
+type clientImpl struct {
```

Comment:
Is the lower case intended? It looks inconsistent with other types.

Reply:
I assume it's this way because it's private.

Reply:
Yes, that's right. It's indicative of a bigger problem with this change, though, which is what I'm working on now.